samtranslator/feature_toggle/feature_toggle.py

```diff
@@ -19,12 +24,35 @@ class FeatureToggle:
     SAM is executing or not.
     """
 
+    DIALUP_RESOLVER = {
+        "toggle": ToggleDialup,
+        "account-percentile": SimpleAccountPercentileDialup,
+    }
+
     def __init__(self, config_provider, stage, account_id, region):
         self.feature_config = config_provider.config
         self.stage = stage
         self.account_id = account_id
         self.region = region
 
+    def _get_dialup(self, region_config, feature_name):
+        """
+        get the right dialup instance
+        if no dialup type is provided or the specified dialup is not supported,
+        an instance of NeverEnabledDialup will be returned
+
+        :param region_config: region config
+        :param feature_name: feature_name
+        :return: an instance of
+        """
+        dialup_type = region_config.get("type")
+        if dialup_type in FeatureToggle.DIALUP_RESOLVER:
```

Comment:
is FeatureToggle.DIALUP_RESOLVER.keys() required?

Reply:
no, just using `if key in dict_obj:` is good